src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {
+        person.age
+        && (
+          <p className="Person__age">
+            {`I am ${person.age}`}
+          </p>
+        )
+      }
+    <p className="Person__partner">
+      {
+        person.isMarried
+          ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Let's create a variable for wife/husband to avoid nesting of ternary operators
![image](https://github.com/mate-academy/react_person/assets/96409563/3cd6cb04-0eec-4a2b-8470-5d4864799075)